docs/introduction.md

```diff
@@ -138,6 +138,43 @@ ncn-m001# sat bash
 (CONTAINER_ID) sat-container# sat status
 ```
 
+## SAT in CSM 1.3
+
```

Comment:
Line 29 of this file is incorrect for systems running CSM 1.3. I'd suggest updating it to something like the following:

In CSM 1.3 and newer, the `sat` command is automatically available on all the Kubernetes NCNs. See [SAT in CSM] (introduction.md#sat-in-csm) for more information. Older versions of CSM do not have the `sat` command automatically available, and SAT must be installed as a separate product as part of the HPE Cray EX System base installation

Reply:
Thanks, addressed in 9609bb3c912e676d49f5569bee883d6c1dd33634.